app/assets/javascripts/admin/addon/templates/emojis-index.hbs

```diff
@@ -0,0 +1,70 @@
+<AdminPageSubheader @titleLabel="admin.emoji.title">
+  <:actions as |actions|>
+    <actions.Primary @route="adminEmojis.new" @label="admin.emoji.new" />
+  </:actions>
+</AdminPageSubheader>
+
+{{#if this.sortedEmojis}}
+  <div class="inline-form">
+    <label class="label">{{i18n "admin.emoji.filter_label"}}</label>
+    <ComboBox
+      @value={{this.filter}}
+      @content={{this.sortingGroups}}
+      @nameProperty={{null}}
+      @valueProperty={{null}}
+      @onChange={{action "filterGroups"}}
+    />
+  </div>
+
+  <table id="custom_emoji" class="d-admin-table">
+    <thead>
+      <tr>
+        <th>{{i18n "admin.emoji.image"}}</th>
+        <th>{{i18n "admin.emoji.name"}}</th>
+        <th>{{i18n "admin.emoji.group"}}</th>
+        <th colspan="3">{{i18n "admin.emoji.created_by"}}</th>
+      </tr>
+    </thead>
+    <tbody>
+      {{#each this.sortedEmojis as |e|}}
+        <tr class="d-admin-row__content">
+          <td class="d-admin-row__overview">
+            <img
+              class="emoji emoji-custom"
+              src={{e.url}}
+              title={{e.name}}
+              alt={{i18n "admin.emoji.alt"}}
+            />
+          </td>
+          <td class="d-admin-row__detail">
+            <div class="d-admin-row__mobile-label">
+              {{i18n "admin.emoji.name"}}
+            </div>
+            :{{e.name}}:
+          </td>
+          <td class="d-admin-row__detail">
+            <div class="d-admin-row__mobile-label">
+              {{i18n "admin.emoji.group"}}
+            </div>
+            {{e.group}}
+          </td>
+          <td class="d-admin-row__detail">
+            <div class="d-admin-row__mobile-label">
+              {{i18n "admin.emoji.created_by"}}
+            </div>
+            {{e.created_by}}
+          </td>
+          <td class="d-admin-row__controls action">
+            <DButton
+              @action={{fn this.destroyEmoji e}}
+              @icon="trash-can"
+              class="btn-danger"
```

Comment:
Should be `btn-small` as well

Reply:
![](https://media0.giphy.com/media/v1.Y2lkPTc5MGI3NjExYXpuejE2YWVhNWZtZGpjcDVmcTl1aTg4N3ljbGZpcmNkZTd1d2xwcyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/3ohc11UljvpPKWeNva/giphy.webp)